src/main/java/io/debezium/connector/vitess/VitessDatabaseSchema.java

```diff
@@ -64,12 +64,14 @@ private boolean isFilteredOut(TableId id) {
     }
 
     /** Refreshes the schema content with a table constructed externally */
-    public void refresh(Table table) {
+    @Override
```

Comment:
This is not properly formatted

Reply:
Weird. I could swear I had done it. Force-pushed now.